api/lib/domain/models/AnswerCollectionForScoring.js

```diff
@@ -38,6 +46,10 @@ module.exports = class AnswerCollectionForScoring {
     return numberOfNonNeutralizedChallenges;
   }
 
+  numberOfNeutralizedChallenges() {
+    return this.answers.reduce((count, answer) => count + (answer.isNeutralized() ? 1 : 0), 0);
```

Comment:
C'est le genre de truc ou un lodash sumBy apporte un peu de lisibilité : 
```
const thingsToBeCountedIfValueIsTrue = [{value: true}, {value: true}, {value: false}];
_.sumBy(thingsToBeCountedIfValueIsTrue, (thingToBeCountedIfValueIsTrue) => thingToBeCountedIfValueIsTrue.value)
```

et dans le cas où on voudrais pas utiliser lodash on peut le faire en deux opérations:
`
this.answers.filter((answer) => answer.isNeutralized()).length`

Le reduce c'est assez souvent peu lisible...